contracts/diaspore/DebtEngine.sol

```diff
@@ -35,6 +35,11 @@ contract DebtEngine is ERC721Base {
         uint256 _tokens
     );
 
+    event ReadedOracle(
```

Comment:
Maybe we can name this event RadedOracleBatch and emit how many loans are going to use this Oracle rate